### PR TITLE
feat: added beak

### PR DESCRIPTION
**Pull request by 0xdeafcafe**
### Checklist

- [x] I have read and completed the checklist of [Submission Guide](https://github.com/electron/apps/blob/master/contributing.md#submission-guidelines)
- [x] My application is open-source and has been created for at least 20 days
- [ ] My application is closed-source and here is a proof it uses Electron:

### Description

Beak is an electron based cross-platform tool which makes building, testing, and reverse engineering API's easier. The website is over at [getbeak.app](https://getbeak.app).

peek

### CODE REVIEW

**Comment by 0xdeafcafe** [comment]
I think this is unblocked now?

**Review by BlackHole1 — APPROVED** [review]
LGTM

File: apps/beak/beak.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - api-client
+  - api-testing
+  - api-building
+  - http
+  - https
+  - testing
+  - privacy
+category: Developer Tools
+screenshots:
+  - imageUrl: https://github.com/getbeak/beak/blob/master/assets/home-dark.png?raw=true
```
Comment (BlackHole1) [inline apps/beak/beak.yml:16]: Can you add an empty line at the content end?

Comment (0xdeafcafe) [reply apps/beak/beak.yml]: @BlackHole1 Done :)

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**